clients/src/main/resources/common/message/ConsumerGroupHeartbeatRequest.json

```diff
@@ -43,21 +43,6 @@
       "about": "null if it didn't change since the last heartbeat; the subscribed topic regex otherwise" },
     { "name": "ServerAssignor", "type": "string", "versions": "0+", "nullableVersions": "0+", "default": "null",
       "about": "null if not used or if it didn't change since the last heartbeat; the server side assignor to use otherwise." },
-    { "name": "ClientAssignors", "type": "[]Assignor", "versions": "0+", "nullableVersions": "0+", "default": "null",
```

Comment:
For anyone who was initially confused like me who may read this -- we can remove the fields because this API is still unstable/unreleased.

Reply:
I hope it's ok to slide in with an informal question here. I'm also curious: what's the source of truth for API stability?

Is it something we could work on adding as an annotation to the schema spec files?

Reply:
It is an annotation. See in the spec

```
"latestVersionUnstable": true,
```

Reply:
Oh my, now I feel dumb 😄 Thanks for pointing this out!